kde-misc/kio-admin/kio-admin-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+ECM_TEST="false"
```

Comment:
unnecessary, default in `ecm.eclass`

Reply:
Also removed ECM_HANDBOOK as I realised DocTools wasnt present.